.github/workflows/auto-comment.yml

```diff
@@ -0,0 +1,16 @@
+name: Zowe CLI Auto Responder for New Issues
+# This workflow is triggered when a label is added to an issue.
+on:
+  issues:
+    types: labeled
+
+jobs:
+  processLabelAction:
+    name: Process Label Action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Process Label Action
+        uses: hramos/label-actions@v1
```

Comment:
I believe I've seen `hramos/respond-to-issue-based-on-label@v2` been used in other repos.
@awharn, any thoughts?